frontend/src/Components/NewCharts/LineChart.js

```diff
@@ -96,15 +99,19 @@ export default function LineChart({
 
   return (
     <>
-      <div ref={setReferenceElement} />
-      <Tooltip
-        ref={setPopperElement}
-        style={{ ...styles.popper, width: '300px' }}
-        {...attributes.popper}
-      >
-        {tooltipText}
-      </Tooltip>
-      <Line options={options} data={data} />
+      {displayStopPurposeTooltips && (
+        <>
+          <div ref={setReferenceElement} />
+          <Tooltip
```

Comment:
This popper tooltip was re-rendering on scroll, which caused the graphs to re-render and provide a bad experience. Here we just want to show the tooltip if we actually need to.